src/Server.ts

```diff
@@ -10,6 +11,7 @@ class Server{
   }
```

Comment:
Changes your class Server as I defined below : -


class Server {
  public app: express.Express;
  
  constructor (private config: IConfig) {
    this.app = express();
  }